### DISCUSSION TITLE
Homematic IP local taucht nicht in Integrationen auf

### DISCUSSION THREAD

**Discussion by Bomaker** [OpenCCU/OpenCCU#2517] [🇩🇪 Fragen&Antworten] [answered]
Hallo Alle,

Bin https://github.com/jens-maus/RaspberryMatic/wiki/HomeAssistant-Integration relativ weit gekommen. Raspberrymatic läuft, Geräte sind sichtbar. 

Ich scheitere an Schritt 7. "Install 'Homematic(IP) Local' integration". Obwohl ich das Custom Repository aus Schritt 4. installiert habe, taucht es nicht auf. Bei "Integration hinzufügen" sehe ich nur ein altes Homematic Logo und Homematic IP Cloud.

Wo ist mein Fehler?

Danke im Voraus!

**Comment by Bomaker** [comment]
Es entsteht wird kein '/config/custom_components/homematicip_local' erstellt.

**Reply by Bomaker** [reply]
Grund gefunden. Schritt 5 nicht gemacht (weil schon ein HACS da war). Das war aber zu alt und vor einem Update-Hemmnis, siehe
https://github.com/hacs/integration/issues/2553
Die Magie ist wget -O - https://get.hacs.xyz | bash -
Jetzt hänge ich an Schritt 9

**Reply by Bomaker** [reply, marked as answer]
Nur falls jemand mitliest - es klappt, wenn man einen neuen ser mit Admin-Rechten in Raspberrymatic anlegt und benutzt.

**Reply by Marcma1964** [reply]
Leider klappt es bei mir auch nicht mit einem neuen User mit Admin Rechten. Sehr schade. Bekomme es einfach nicht zum laufen:

VALIDATE_CONFIG_AND_GET_SYSTEM_INFORMATION failed for client BidCos-RF: OSError on temporary_instance-BidCos-RF: (111, 'Connection refused')
VALIDATE_CONFIG_AND_GET_SYSTEM_INFORMATION failed for client HmIP-RF: OSError on temporary_instance-HmIP-RF: (111, 'Connection refused')
